accelerator/config.h

```diff
@@ -72,6 +72,8 @@ extern "C" {
 #define MAM_FILE_PREFIX "/tmp/mam_bin_XXXXXX"
 #define BUFFER_LIST_NAME "txn_buff_list"
 #define DONE_LIST_NAME "done_txn_buff_list"
+#define MAM_BUFFER_LIST_NAME "mam_buff_list"
+#define MAM_DONE_LIST_NAME "done_mam_buff_list"
```

Comment:
What is MAM done?

Reply:
I use another redis list which named as `done_mam_buff_list` to save the MAM requests which has been broadcasted.

Reply:
How about calling it "complete list?"

Reply:
sure